pkg/specs/chart.go

```diff
@@ -390,3 +376,91 @@ func (r *Resolver) maybeSplitMultidocYaml(ctx context.Context, localPath string)
 
 	return nil
 }
+
+type ListK8sYaml struct {
+	APIVersion string        `json:"apiVersion" yaml:"apiVersion"`
+	Kind       string        `json:"kind" yaml:"kind" hcl:"kind"`
+	Items      []interface{} `json:"items" yaml:"items"`
+}
+
+func (r *Resolver) maybeSplitListYaml(ctx context.Context, path string) error {
+	files, err := r.FS.ReadDir(path)
+	if err != nil {
+		return errors.Wrapf(err, "read files in %s", path)
+	}
+
+	var lists []state.List
+	for _, file := range files {
+		filePath := filepath.Join(path, file.Name())
+
+		if file.IsDir() {
+			// TODO: handling nested list yamls
```

Comment:
not sure of how important this is / how often it will come up

Reply:
Perhaps call the function recursively and then `continue`? Either way, I know you need to add a `continue` here or things will blow up when it hits the ReadFile call below